python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -1400,7 +1400,7 @@ def _get_entity_type_whitelist(self, project_id, config_descriptor):
             # environment. Those files contain entity type names that are lower cased,
             # so it's easiest just to do everything that way.
             type_whitelist = set(
-                [t.lower() for t in constants.BASE_ENTITY_TYPE_WHITELIST]
+                [t.lower() for t in constants.BASE_ENTITY_TYPE_ALLOW_LIST]
```

Comment:
Thanks for taking the time to update the variable name. We really appreciate it.